packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -126,6 +134,125 @@ export const createRDSInstance = async (
   }
 };
 
+/**
+ * Creates a new RDS Aurora serverless V2 cluster with one DB instance using the given input configuration.
+ * @param config Configuration of the database cluster. If password is not passed an RDS managed password will be created.
+ * @returns EndPoint address, port and database name of the created RDS cluster.
+ */
+export const createRDSCluster = async (
+  config: RDSConfig,
+): Promise<{
+  clusterArn: string;
+  endpoint: string;
+  port: number;
+  dbName: string;
+  password: string;
+  managedSecretArn: string;
+  dbInstance: DBInstance;
```

Comment:
Consider a named structure (e.g., a typealias or interface) that encompasses these fields, to centralize the definition and make future maintenance easier.